.github/workflows/flask.yml

```diff
@@ -0,0 +1,67 @@
+name: Django CI
+
+on:
+  push:
+    branches: [master, testing]
+  pull_request:
+    branches: [master, testing]
+ 
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Is it better to use Debian?

Reply:
This is related to GitHub infraestructure.